src/jit/codegenlegacy.cpp

```diff
@@ -15387,7 +15387,14 @@ void CodeGen::genTableSwitch(regNumber reg, unsigned jumpCnt, BasicBlock** jumpT
 
 #endif // _TARGET_ARM_
 
-    if (jumpCnt < minSwitchTabJumpCnt)
+    if (jumpCnt < minSwitchTabJumpCnt ||
+#ifdef PLATFORM_UNIX
+        // Force using an inlined jumping instead switch table generation
+        // See https://github.com/dotnet/coreclr/issues/13194
+        (compiler->IsTargetAbi(CORINFO_CORERT_ABI)))
+#else
+        0)
+#endif
     {
```

Comment:
Use `_TARGET_UNIX_`, not `PLATFORM_UNIX`. Also, the bug says this issue is only for ARM. Also, please don't simply reference the bug: write the full description here, as a comment, for why this case exists. Thus, I would expect to see:

```
    bool useJumpSequence = jumpCnt < minSwitchTabJumpCnt;

#if defined(_TARGET_UNIX_) && defined(_TARGET_ARM_)
    // Force using inlined jumping instead switch table generation.
    // << insert full description of WHY here >>     
    // See also [https://github.com/dotnet/coreclr/issues/13194](https://github.com/dotnet/coreclr/issues/13194).
    useJumpSequence = useJumpSequence || compiler->IsTargetAbi(CORINFO_CORERT_ABI);
#endif // defined(_TARGET_UNIX_) && defined(_TARGET_ARM_)

    if (useJumpSequence)
    ...
```